extensions/amp-lightbox-gallery/0.1/service/lightbox-manager-impl.js

```diff
@@ -36,8 +36,9 @@ const LIGHTBOX_ELIGIBLE_TAGS = {
   'amp-facebook': true,
 };
 
-const ELIGIBLE_TAP_TAGS = {
+export const ELIGIBLE_TAP_TAGS = {
```

Comment:
This looks like it's duplicated between the lightbox manager and the `amp-image-viewer`. Will these always need to be in sync, or are they semantically different lists? If they need to be synced, they should either be stored and accessed from a common location, or there should be a comment instructing future editors to make sure changes are reflected in both maps.

Reply:
These are not necessarily the same and should not be sync-ed. `ELIGIBLE_TAP_TAGS` for `<amp-lightbox-gallery>` is everything that we can open up on tap. This in the future may include `<div>`s, `<svg>`, etc. anything we want to be able to lightbox. 

`<amp-image-viewer>`'s `ELIGIBLE_TAGS` should include anything that we can zoom and pan within an image viewer. I believe this is only going to be `<amp-anim>` and `<amp-img>`. Since `<amp-image-viewer>` will eventually launch as it's own component, I don't think we should make that dependency. It's possible that for all practical purposes that they will be the same, but supporting a component in one does not mean it will work without custom logic for the other.

Reply:
Cool, sounds good then!